Controllers/Frontend/Adyen.php

```diff
@@ -144,7 +144,7 @@ private function paymentDetails(string $post, string $detail)
         $this->Front()->Plugins()->ViewRenderer()->setNoRender();
 
         $postData = $this->Request()->getPost();
-        $threeDsDetail = (string) $postData['details'][$detail] ?? '';
+        $threeDsDetail = (string) ($postData['details'][$detail] ?? $postData['details'][$post] ?? '');
```

Comment:
@acampos1916 
This is BC compatible way for fetching data from web components library data, 
before: underscore, new (dot).
Example: `threeds2_fingerprint` -> `threeds2.fingerprint`